README.md

```diff
@@ -457,13 +457,34 @@ export default Controller.extend({
 });
 ```
 
+
 ```hbs
 {{! application/template.hbs}}
 {{dummy-form changeset=(changeset model (action "validate"))}}
 ```
 
 Your action will receive a single POJO containing the `key`, `newValue`, `oldValue` and a one way reference to `changes`.
 
+## Handling Server Errors
+
+When you run `changeset.save()`, under the hood this executes the changeset, and then runs the save method on your original content object, passing its return value back to you.  You are then free to use this result to add additional errors to the changeset if applicable.
+
+For example, if you are using Ember Data and would like to add any model attribute errors that may result from the save back to the changeset:
+
+```js
+changeset.save()
+  .then(() => { console.log('Success!'); })
+  .catch(() => {
+    const errors = this.get('model.errors');
+    errors.forEach(error => {
+      let key = error.attribute;
+      let value = changeset.get(key);
```

Comment:
I wouldn't pass `value` in here, it wouldn't make any sense since `get` would only get a valid value. Just the `validation` itself is fine